HW02_B_Mimic_starter.Rmd

```diff
@@ -52,6 +62,17 @@ data("iris")
 
 Using the iris dataset, make this graph:
 ```{r graph 2 code, echo=FALSE}
+#Re-order the values of Species 
+iris$Species <- factor(iris$Species,levels=c("versicolor","setosa","virginica"))
+ggplot(iris,aes(Sepal.Length, Petal.Length, shape=Species, color=Species)) + 

```

Comment:
I did not really notice that the order was wrong. Clever and rational way to change the order.